src/components/collection-card/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { IconCollections16 } from '@hashicorp/flight-icons/svg-react/collections-16'
+import CardLink from 'components/card-link'
+import {
+  CardEyebrow,
+  CardHeading,
+  CardBody,
+  CardBadges,
+} from 'components/tutorial-collection-cards'
+import { CollectionCardProps } from './types'
+import s from './collection-card.module.css'
+
+/**
+ * Render a card that links to a collection.
+ *
+ * TODO: note that not all collection links will work, for now.
```

Comment:
This should be finished, it was handled via #267 👌 So long as we use the `getCollectionSlug` helper function, it will be handled properly.

Reply:
oh cool nice! 🤩 🎉  I'll drop this comment, had just kept it cause it was in the previous stubbed collection card in `tutorial-view`